### PR TITLE
Adding toastr notifications with new "SuccessfullySaved" localization

### LINKED ISSUE #19324: Toaster should display after create, update, delete operations
### Documentation

- Localization enhancements are required.
- Toaster messages should be displayed for add, update, or delete operations within feature management, identity, and tenant management modules.

[screen-capture.webm](https://github.com/abpframework/abp/assets/81169996/1291243f-cf3a-4466-9535-c87a1ec2d541)

### PR DESCRIPTION
**Pull request by yusuf-cirak**
### Description

Resolves #19324

- Adding localization messages.
- Adding toast messages to operations in feature-management, identity, tenant-management modules.

### Checklist

- [x] I fully tested it as developer

### How to test it?

- Perform some create or update operations.

### CODE REVIEW

**Review by masum-ulu — APPROVED** [review]

File: npm/ng-packs/packages/permission-management/src/lib/components/permission-management.component.ts [hunk 1]
```diff
@@ -117,7 +118,11 @@ export class PermissionManagementComponent
 
   trackByFn: TrackByFunction<PermissionGroupDto> = (_, item) => item.name;
 
-  constructor(protected service: PermissionsService, protected configState: ConfigStateService) { }
+  constructor(
+    protected service: PermissionsService,
+    protected configState: ConfigStateService,
+    private readonly toasterService: ToasterService,
```
Comment (masum-ulu) [inline npm/ng-packs/packages/permission-management/src/lib/components/permission-management.component.ts]: Mark as `protected`

**Review by oykuermann — CHANGES_REQUESTED** [review]
Localization names should be 'SavedSuccesfully' and 'DeletedSuccesfully'. Please, rewrite all the localizations.
image

**Comment by yusuf-cirak** [comment]
> Localization names should be 'SavedSuccesfully' and 'DeletedSuccesfully'. Please, rewrite all the localizations. image

I'm not sure about this. Other UI's will need to change their localizations too. If that's okay, i can change all.

**Comment by gizemmutukurt** [comment]
This is how the majority approved it on the channel @yusuf-cirak 

image

**Review by masum-ulu — APPROVED** [review]

**Review by oykuermann — APPROVED** [review]

**Pull request merged**